<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="nextFrameButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>440</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previousFrameButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>440</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous Frame</string>
    </property>
   </widget>
   <widget class="QLabel" name="transformationsLabel">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>230</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Transformations</string>
    </property>
   </widget>
   <widget class="pixeldisplay" name="mainDrawing" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>360</width>
      <height>360</height>
     </rect>
    </property>
   </widget>
   <widget class="pixeldisplay" name="frameDisplay" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>420</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="pixeldisplay" name="animationDisplay" native="true">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="fpsSlider">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>190</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="reflectVerticalButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>250</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Vert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reflectHorizontalButto">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>250</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Horiz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>250</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </widget>
   <widget class="QLabel" name="fpsLabel">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>190</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>12</string>
    </property>
   </widget>
   <widget class="QLabel" name="fpsConstant">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FPS</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>370</y>
      <width>120</width>
      <height>141</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>118</width>
       <height>139</height>
      </rect>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>80</height>
       </rect>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>101</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="buttonLayout"/>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="ColorLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QLabel" name="colorDisplay">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Set Transparency</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="addFrameButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>450</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteFrameButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>450</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Frame</string>
    </property>
   </widget>
   <widget class="QLabel" name="layersLabel">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>350</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Layers</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>120</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QPushButton" name="brushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>100</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>icons/brush.png</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="eraserButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>100</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>icons/eraser.png</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="paintBucketButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>100</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>icons/paint-bucket.png</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="eyeDropperButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>100</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>icons/eyedropper.png</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="addLayerButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>340</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteLayerButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>340</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QWidget" name="mouseListener" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>360</width>
      <height>360</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Transparency</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionLoad"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save Sprite</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Load an ssp file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pixeldisplay</class>
   <extends>QWidget</extends>
   <header>pixeldisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
